### PR TITLE
Html report improvements

### PR DESCRIPTION
**Pull request by zhostasa**
Verticalized HTML Report tables
Added tooltip to clarify highlights
"Gross throughput" changed to "Troughput w/ errors" and hidden in reports without errors (NG in Chrome)

### CODE REVIEW

**Comment by mgencur** [comment]
I wanted to integrate this PR but ran into https://github.com/radargun/radargun/issues/394 when generating the report. Will wait for the resolution.

File: reporters/reporter-default/src/main/java/org/radargun/reporting/html/HtmlReporter.java [hunk 1]
```diff
@@ -207,8 +207,10 @@ private void writeIndexDocument(Collection<Report> reports) {
       index.prepareServiceConfigs(reports);
 
       try {
-         copyResources("style.css");
-         copyResources("script.js");
+         copyResources("html/templates/", "style.css");
+         copyResources("html/templates/", "script.js");
+         copyResources("html/icons/", "ic_arrow_drop_down_black_24dp.png");
+         copyResources("html/icons/", "ic_arrow_drop_up_black_24dp.png");
```
Comment (jmarkos) [inline reporters/reporter-default/src/main/java/org/radargun/reporting/html/HtmlReporter.java]: These icons seem missing.

File: reporters/reporter-default/src/main/resources/html/templates/testReport.ftl [hunk 2]
```diff
@@ -124,8 +158,8 @@
             	   	<th>std.dev</th>
 	
     	           	<#if operationData.getPresentedStatistics()?seq_contains(StatisticType.OPERATION_THROUGHPUT) >
-        	        	<th>gross operation throughput</th>
-            	    	<th>net operation throughput</th>
+            	    	<th>Troughput</th>
```
Comment (jmarkos) [inline reporters/reporter-default/src/main/resources/html/templates/testReport.ftl]: typo - should be Throughput

File: reporters/reporter-default/src/main/java/org/radargun/reporting/html/HtmlReporter.java (hunk 1 above)
Comment (zhostasa) [inline reporters/reporter-default/src/main/java/org/radargun/reporting/html/HtmlReporter.java]: fixed

**Comment by zhostasa** [comment]
Squashed this whole thing, I am afraid it might be a bit unwieldy now, sorry for that

**Comment by mgencur** [comment]
@alanfx , did you want to look at this PR? Can you please integrate this soon?

**Comment by jmarkos** [comment]
Not a fan of the squashing, but could you please at least end the sentences with a dot so the commit message is readable? Command-line git removes the newline characters.

**Comment by zhostasa** [comment]
Good point, will remember to

**Comment by rvansa** [comment]
@jmarkos I don't think that command-line git removes newlines (`git log` usually displays them just fine), but @zhostasa the convention is to have name of the commit, blank line, and then use stars or dashes for list items (if the text is not a paragraph).

**Comment by jmarkos** [comment]
Ah right, I use some custom commands for log viewing and they cannot handle it... Sorry for confusion.

**Comment by zhostasa** [comment]
That is a convention I was not aware of (quite unsurprisingly), but it does explain things (like git using the first line as commit name in the pull request), thanks:-)

**Comment by alanfx** [comment]
OK, the code looks good to me, @zhostasa. Are there any changes still needed, or should I integrate this PR? @mgencur, @rvansa, or @jmarkos?

**Comment by mgencur** [comment]
I'm OK with integrating this.

**Comment by alanfx** [comment]
Integrated. Thanks @zhostasa!

**Pull request closed without merging**